src/api/spec/cassettes/Webui_Cloud_Azure_ConfigurationsController/DELETE_destroy/1_3_1.yml

```diff
@@ -1,124 +0,0 @@
----
```

Comment:
I'm not totally sure that's correct, I've tried to follow the [Guidelines](https://github.com/openSUSE/open-build-service/wiki/Testing-with-VCR)